sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/models/ReadOptions.java

```diff
@@ -22,4 +24,12 @@ public static ReadOptions CreateInfiniteCacheOption() {
     public static ReadOptions CreateNoCacheOption() {
         return new ReadOptions(BaseOptions.NO_CACHE);
     }
+
+    /**
+     * @param expiredAt timestamp of when the document is expired.
+     * @return whether a document is expired.
+     */
+    public static boolean isExpired(long expiredAt) {
```

Comment:
Move it here, because I do not think writeOptions need isExpired method, or I am wrong 🙈

Reply:
and after discuss with @dhei not sure keep in ReadOptions, or put it into util.java file :D